homeassistant/components/velbus/config_flow.py

```diff
@@ -1,5 +1,6 @@
 """Config flow for the Velbus platform."""
-import velbus
+import velbusaio
+from velbusaio.exceptions import VelbuConnectionFailed
```

Comment:
Is it intended that the exception is named `VelbuConnectionFailed` instead of `VelbusConnectionFailed`?

Reply:
no ...